python/lsst/ip/diffim/imagePsfMatch.py

```diff
@@ -281,6 +281,10 @@ def __init__(self, *args, **kwargs):
         PsfMatchTask.__init__(self, *args, **kwargs)
         self.kConfig = self.config.kernel.active
         self._warper = afwMath.Warper.fromConfig(self.kConfig.warpingConfig)
+        # the background subtraction task uses a config from an unusual location,
+        # so cannot easily be constructed with makeSubtask
```

Comment:
I think ip_diffim needs significant refactoring, but it is out of scope for this ticket.